worker/containerd/containerd.go

```diff
@@ -28,6 +29,10 @@ import (
 // NewWorkerOpt creates a WorkerOpt.
 func NewWorkerOpt(root string, address, snapshotterName, ns string, rootless bool, labels map[string]string, dns *oci.DNSConfig, nopt netproviders.Opt, apparmorProfile string, parallelismSem *semaphore.Weighted, traceSocket string, opts ...containerd.ClientOpt) (base.WorkerOpt, error) {
 	opts = append(opts, containerd.WithDefaultNamespace(ns))
+	if runtime.GOOS == "freebsd" {
+		opts = append(opts, containerd.WithDefaultRuntime("wtf.sbk.runj.v1"))
```

Comment:
Could you create docs/freebsd.md to describe how to set up these dependencies?

Reply:
Good idea, thanks!

Reply:
It should be as simple as `pkg install runj containerd` now?

Reply:
Almost there.

We need to update the git snapshot of the runj port to include https://github.com/samuelkarp/runj/pull/31

@arrowd if you could help drive this, that would be amazing!

Reply:
Done in https://cgit.freebsd.org/ports/commit/?id=101e87b72b63916ee63a2c8d9fb8abad45c4cb6b

Reply:
Perfect, thanks! I'll wait for `pkg+http://pkg.FreeBSD.org/${ABI}/latest` to update, and then I'll update  `Vagrantfile`. That will simplify CI a lot.

I also updated `docs/freebsd.md`. So yes, it should be as simple as installing runj & containerd.